### ISSUE TITLE
Checkboxes are not being checked when using vuetify 

### ISSUE THREAD

**Issue by tscgist** [eclipsesource/jsonforms-vuetify-renderers#42]
Pre-selection of checkboxes is working when using ui-schema in vuetify context.
Selecting and deselecting works on mouse click.

Without using ui-schema checkboxes are pre-selected correclty.

I recognized that element "<div class="v-input--selection-controls__ripple primary--text"></div>" is using class "primary--text" for checked states. This class is not set initially.

**Comment by sdirix** [comment]
Hi @tscgist, can you elaborate what exactly the problem is? If possible, please provide a step-by-step guide to reproduce the problem.

**Comment by davidpede** [comment]
I've come across the same issue. The following reproduces the issue:

Schema:
```
{
  "type": "object",
  "properties": {
    "active": {
      "type": "boolean"
    }
  }
}
```
UI Schema:
```
{
  "type": "VerticalLayout",
  "elements": [
    {
      "type": "Control",
      "scope": "#/properties/active"
    }
  ]
}
```
Data:
```
{
  "active": true
}
```

The checkbox is not selected when the form is initialised.

**Comment by sdirix** [comment]
Fixed with #51
